pcap4j-core/src/main/java/org/pcap4j/core/NativeMappings.java

```diff
@@ -119,6 +119,9 @@ static native  Pointer pcap_open_live(
   // pcap_t *pcap_open_offline(const char *fname, char *errbuf)
   static native Pointer pcap_open_offline(String fname, PcapErrbuf errbuf);
 
+  // pcap_t *pcap_open_offline_with_tstamp_precision(const char *fname, u_int precision, char*errbuf);
+  static native Pointer pcap_open_offline_with_tstamp_precision(String fname, int precision, PcapErrbuf errbuf);
```

Comment:
It seems pcap_open_offline_with_tstamp_precision is a very new api and most libpcap distributions don't implement it. Direct mapping is not suitable for such an api. Instead, please use interface mapping like pcap_set_rfmon.